src/Dependencies/Collections/Microsoft.CodeAnalysis.Collections.Package.csproj

```diff
@@ -2,7 +2,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
❓ Is there a need to change this? This project is usable outside Roslyn.

Reply:
The reason for this is I'm also gonna bump our BCL reference to 7, which would cause error for project targeting 3.1

>packages/system.text.encoding.codepages/7.0.0/buildTransitive/netcoreapp2.0/System.Text.Encoding.CodePages.targets(4,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) System.Text.Encoding.CodePages 7.0.0 doesn't support netcoreapp3.1 and has not been tested with it. Consider upgrading your TargetFramework to net6.0 or later. You may also set <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings> in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk.

Reply:
> You may also set true in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk.

Can we set something to true to ignore the error? This is a source package anyways, so it shouldn't matter.

Reply:
Sure. I just wasn't aware of a reason that warrants suppressing the warning.